examples/webgl_loader_pdb.html

```diff
@@ -137,46 +115,26 @@
 
 				//
 
-				loadMolecule( 'models/pdb/caffeine.pdb' );
-				createMenu();
+				loadMolecule( params.molecule );
 
 				//
 
 				window.addEventListener( 'resize', onWindowResize );
 
-			}
-
-			//
-
-			function generateButtonCallback( url ) {
-
-				return function () {
-
-					loadMolecule( url );
-
-				};
-
-			}
-
-			function createMenu() {
-
-				for ( const m in MOLECULES ) {
-
-					const button = document.createElement( 'button' );
-					button.innerHTML = m;
-					menu.appendChild( button );
-
-					const url = 'models/pdb/' + MOLECULES[ m ];
+				//
 
-					button.addEventListener( 'click', generateButtonCallback( url ) );
+				const gui = new GUI();
 
-				}
+				gui.add( params, 'molecule', MOLECULES ).onChange( loadMolecule );
```

Comment:
I think instead of `'molecule'` it should be `'compound'`?

Reply:
Should I update then all places where `molecule` is used as a term? 

BTW: I think something like Ethanol is actually a molecule^^. `compound` seems to be just a synonym.

Reply:
Oh true, they're molecules. I got confused sorry 😅